### PR TITLE
Revising subscription flow to fit new backend functionaltiy

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[**PT-Story:**  ](https://www.pivotaltracker.com/story/show/178256034)
### User Story 
``` 
As a dev team
In order to have matching functionality with the backend
We need to revise Stripe functionality to use payment methods instead of tokens
``` 
## Changes proposed in this pull request: 
* Revises createToken() into createPaymentMethod()
* Adds subscription plans

## What I have learned working on this feature: 
* New Stripe functionality with createPaymentMethod()

### CODE REVIEW

File: cypress/integration/userCanRegisterAnAccountAndSubscribe.feature.js [hunk 1]
```diff
@@ -93,20 +94,10 @@ describe('User is able to register an account and subscribe', () => {
     });
   });
 
-  describe('unsuccessfully with faulty subcription request', () => {
+  describe.only('unsuccessfully with faulty subcription request', () => {
```
Comment (1CIM) [inline cypress/integration/userCanRegisterAnAccountAndSubscribe.feature.js]: Should this still be here?

**Review by christerforsgren91 — APPROVED** [review]
Good job!

File: src/components/Register.jsx [hunk 2]
```diff
@@ -21,109 +22,171 @@ const Register = () => {
...
         {location.state && (
           <h1
             style={{ marginBottom: 50, color: 'white' }}
             data-cy='subscribe-message'>
             You tried to access a premium article, please subscribe
           </h1>
         )}
+        {!subscriptionPlan && (
+          <>
+            <Card.Group centered>{subscriptionPlanCardList}</Card.Group>
```
Comment (offcast-dev) [inline src/components/Register.jsx:101]: Does Card.Group works if you dont use Cards?

**Review by offcast-dev — APPROVED** [review]

File: src/components/Register.jsx (hunk 2 above)
Comment (LucasKnudsen) [reply src/components/Register.jsx:101]: Apparently, yeah!

**Pull request merged**